organizations/default/client_assets/index.css

```diff
@@ -1 +1,166 @@
-/*  */
+body .input:focus {
+  border-color: #8b8b8b;
+}
+body .input.error {
+  border-color: #d93025;
+}
+body.input {
+  border: 1px solid #ccc;
+}
+
+/* buttons */
+.button {
+  background-color: #333333;
+  color: #ffffff;
+}
+.button:focus {
+  background-color: rgba(0, 0, 0, 0.7);
+}
+.logout .button {
+  background: #ba2121 !important;
+}
+.submit .button {
+  background: #498b26 !important;
+}
```

Comment:
@codesankalp where is this being used?

Reply:
I think it is used in verification page submit button.